### PR TITLE
Add prettier

### LINKED ISSUE #6: Code formatter
Have you a preferred code style for the PR ? Can I add a formater like prettier ?

### PR DESCRIPTION
**Pull request by stex-docto**
fix #6

### CODE REVIEW

**Comment by michelgrootjans** [comment]
This is a big improvement, thank you. However, there are places where I don't agree with prettier, like for example `application/domain/initial-state.js`.

What are the options to be lenient in some places?

**Comment by stex-docto** [comment]
I'll look into it. 👍

File: application/domain/initial-state.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
+// prettier-ignore
```
Comment (stex-docto) [inline application/domain/initial-state.js:1]: This is how you disable prettier for the next instruction only.

**Comment by stex-docto** [comment]
I've put 120 char as line-width and disable formatting on this specific files after refomatting by end to longest line.

**Comment by michelgrootjans** [comment]
Has been merged with #9

**Pull request closed without merging**